src/pspm_split_sessions.m

```diff
@@ -180,20 +181,21 @@
       trimpoint(sn, :) = [splitpoint(sn - 1), max(splitpoint(sn) - 1, 1)];
     end
 
-    if sn > numel(splitpoint)
-      trimpoint(sn, 2) = numel(mrk);
-    else
-      trimpoint(sn, 2) = max(1, splitpoint(sn) - 1);
-    end
-
     if options.suffix == 0
       if trimpoint(sn, 1) == trimpoint(sn, 2) || options.randomITI
-        suffix(sn) = mean(diff(mrk));
+        suffix{sn} = mean(diff(mrk));
       else
-        suffix(sn) = mean(diff(mrk(trimpoint(sn, 1):trimpoint(sn, 2))));
+        suffix{sn}  = mean(diff(mrk(trimpoint(sn, 1):trimpoint(sn, 2))));
       end
     else
-      suffix(sn) = options.suffix;
+      suffix{sn} = options.suffix;
+    end
+    prefix{sn} = options.prefix;
+
+    if sn == 1
+        prefix{sn} = 'none'; % don't trim start for first session
```

Comment:
My understanding is `prefix` and `suffix` are numbers, according to the line 173--174. When they are set to be `none`, my understanding is `none` means "do not perform splitting". In this case, I am not sure if it is good practice to set them to be both `0` to keep them consistent variable types? Just a suggestion.

Reply:
`prefix` and `suffix` are used as arguments for `pspm_trim`. This function uses none to indicate no trimming. So, if we'd want to change the convention this would have to be done in `pspm_trim`. I'd suggest however not making this change within the context of this pull request.